src/App.tsx

```diff
@@ -1,173 +1,64 @@
-                      href="#/people/emma-de-milliano-1876"
-                    >
-                      Emma de Milliano
-                    </a>
-                  </td>
-
-                  <td>
-                    <a href="#/people/emile-haverbeke-1877">
-                      Emile Haverbeke
-                    </a>
-                  </td>
-                </tr>
-
-                <tr data-cy="person" className="has-background-warning">
-                  <td>
-                    <a href="#/people/jan-frans-van-brussel-1761">
-                      Jan Frans van Brussel
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1761</td>
-                  <td>1833</td>
-                  <td>-</td>
-
-                  <td>
-                    <a href="#/people/jacobus-bernardus-van-brussel-1736">
-                      Jacobus Bernardus van Brussel
-                    </a>
-                  </td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/lievijne-jans-1542"
-                    >
-                      Lievijne Jans
-                    </a>
-                  </td>
-
-                  <td>f</td>
-                  <td>1542</td>
-                  <td>1582</td>
-                  <td>-</td>
-                  <td>-</td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a href="#/people/bernardus-de-causmaecker-1721">
-                      Bernardus de Causmaecker
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1721</td>
-                  <td>1789</td>
-
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/livina-haverbeke-1692"
-                    >
-                      Livina Haverbeke
-                    </a>
-                  </td>
-
-                  <td>
-                    <a href="#/people/lieven-de-causmaecker-1696">
-                      Lieven de Causmaecker
-                    </a>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
-          </div>
+import { Navigation } from './components/Loader/pages/Navigation';
+import { PeopleList } from './components/Loader/pages/PeopleList';
+import { Person } from './types';
+
+export const App = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+
+  const loadPeople = () => {
+    getPeople()
+      .then(peopleFromServer => setPeople(peopleFromServer))
+      .catch(() => {
+        setErrorMessage('Something went wrong');
+      });
+  };
+
+  useEffect(() => {
+    loadPeople();
+  }, []);
```

Comment:
```suggestion
  const loadPeople = () => {
    getPeople()
      .then(peopleFromServer => setPeople(peopleFromServer))
      .catch(() => {
        setErrorMessage('Something went wrong');
      });
  };

  useEffect(() => {
    loadPeople();
  }, []);
```
you should do this every time you load your peoplePage
![image](https://user-images.githubusercontent.com/101139968/194119663-dc98e882-59dd-4fbe-970a-cf04892b7451.png)
you should also add loader while you're waiting for response from server
and I feel like async await syntax would be better